brain/thoughts.js

```diff
@@ -1,11 +1,46 @@
+const kaomoji = require('../config/jp.json');
+
+const thoughtMatrix = {
+  worst: ['Angry', 'Crazy', 'Dead', 'Evil', 'Giving Up'],
+  bad: ['Apologizing', 'Hiding', 'Sad', 'Hurt'],
+  neutral: ['Clouds', 'Confused', 'Meh'],
+  good: ['Happy', 'Laughing', 'Hugging'],
+  best: ['Dancing', 'Kissing', 'Excited', 'Love'],
+};
+
+const emojiMatrix = (feel) => {
+  const randKey = Math.floor(Math.random() * Math.floor(thoughtMatrix[feel].length));
+  const kaomojiKey = thoughtMatrix[feel][randKey];
+  const kaomojiObject = kaomoji.categories.find((obj) => obj.name === kaomojiKey);
+  const randEmojiKey = Math.floor(Math.random() * Math.floor(kaomojiObject.entries.length));
+
+  return kaomojiObject.entries[randEmojiKey].emoticon;
+};
+
+const sentimentalThought = (score) => {
```

Comment:
this could also be a good switch statement if you're looking to clean things up

Reply:
is there a way to do it well with logic operators that arent `=` ? i was under the impression it was bad to use switch statements with `< >`

Reply:
oh hm not sure about that one. I know you CAN do it, but maybe you shouldn't for performance reasons or something